doc/changelog.md

```diff
@@ -68,9 +68,6 @@ would no longer decompose the `PSWAP` and `ISWAP` gates to `SWAP`s, `CNOT`s and
 * Remove copies of TOML device configuration files for Lightning device in Catalyst.
   [(#472)](https://github.com/PennyLaneAI/catalyst/pull/472)
 
-* Build and package Catalyst wheels with OpenMP and ZStd.
-  [(#457)](https://github.com/PennyLaneAI/catalyst/pull/457)
-
 * Remove `qextract_p` and `qinst_p` from forced-order primitives.
   [(#469)](https://github.com/PennyLaneAI/catalyst/pull/469)
 
```

Comment:
@josh146 would you fix the following minor MD rendering issue (not related to your changes, just noticed now)?

![image](https://github.com/PennyLaneAI/catalyst/assets/10773383/3107dc94-9816-4982-ae1d-2e69801382f0)

It should be fixed by updating the indentation of line 62 :crossed_fingers: